packages/expect/package.json

```
@@ -12,7 +12,8 @@
   "exports": {
     ".": "./build/index.js",
     "./package.json": "./package.json",
-    "./build/utils": "./build/utils.js"
+    "./build/utils": "./build/utils.js",
+    "./build/matchers": "./build/matchers.js"
```

if we do this, I'd prefer

```suggestion
    "./matchers": "./build/matchers.js"
```

so `build` is not in our public API. That might mess up TS, tho? as of 4.5 something like

```
"./matchers": {
  "types": "./build/matchers.d.ts",
  "default": "./build/matchers.js"
}
```

should work, but I guess we should avoid that up until it's actually out...

---

Yeah when keeping my current implementation typings would work out of the box, with your suggestion not.

I think the current approach is better since TypeScript 4.5 was not yet released - or we use require / ts-ignore for now.

Both approach would work for us, let me know what you think.